python/paddle/distributed/passes/ps_trainer_pass.py

```diff
@@ -434,9 +434,9 @@ def _get_pull_sparse_ops(self, _program, attrs):
             if op.type in SPARSE_OP_TYPE_DICT.keys() \
                     and op.attr('remote_prefetch') is True:
                 param_name = op.input(SPARSE_OP_TYPE_DICT[op.type])[0]
-                if attrs['is_heter_ps_mode']:
-                    # trick for matchnet, need to modify
-                    param_name += op.input("Ids")[0][0]
+                #if attrs['is_heter_ps_mode']:
+                # trick for matchnet, need to modify
```

Comment:
TODO: minxu  trick for matchnet, need to modify

Reply:
@zmxdream how to fix?